pynest/nest/tests/test_NodeCollection.py

```diff
@@ -103,8 +103,11 @@ def test_indexing(self):
         self.assertEqual(n[4], nc_4)
         self.assertEqual(n[-1], nc_4)
         self.assertEqual(n[-3], nc_2)
-        with self.assertRaises(nest.kernel.NESTError):
-            n[7]
+        self.assertEqual(n[-5], nc_0)
+        with self.assertRaises(IndexError):
+            n[5]
+        with self.assertRaises(IndexError):
+            n[-6]
```

Comment:
@Silmathoron checking of out-of-range single index is done here

Reply:
ah, my bad, I was looking only at the new diff!